### PR TITLE
Dev privateendpoint

### PR DESCRIPTION
**Pull request by chinthakaru**
## Description
Updated the Examples with static ip address for storage subresources.

## Type of Change

- [ ] Non-module change (e.g. CI/CD, documentation, etc.)
- [x ] Azure Verified Module updates:
  - [ ] Bugfix containing backwards compatible bug fixes, and I have NOT bumped the MAJOR or MINOR version in `locals.version.tf.json`:
    - [ ] Someone has opened a bug report issue, and I have included "Closes #{bug_report_issue_number}" in the PR description.
    - [ ] The bug was found by the module author, and no one has opened an issue to report it yet.
  - [ ] Feature update backwards compatible feature updates, and I have bumped the MINOR version in `locals.version.tf.json`.
  - [ ] Breaking changes and I have bumped the MAJOR version in `locals.version.tf.json`.
  - [ ] Update to documentation

# Checklist

- [ x] I'm sure there are no other open Pull Requests for the same update/change
- [x ] My corresponding pipelines / checks run clean and green without any errors or warnings
- [ x] I did run all  [pre-commit](https://azure.github.io/Azure-Verified-Modules/contributing/terraform/terraform-contribution-flow/#5-run-pre-commit-checks) checks

### CODE REVIEW

File: examples/private-endpoint-Staticip/main.tf [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+  # Provide static ip address for each endpoint. Make sure to use a unique IP address for each endpoint to avoide conflicts.
+  endpoints = {
+    blob  = "192.168.0.7"
+    queue = "192.168.0.8"
+    table = "192.168.0.9"
+    file  = "192.168.0.10"
+  }
+}
+
+module "public_ip" {
```
Comment (jaredfholgate) [inline examples/private-endpoint-Staticip/main.tf:101]: There ss an AVM module for public ip now, would be better to switch to that if using a module.

File: examples/private-endpoint-Staticip/main.tf [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+  account_tier                  = "Standard"
+  account_kind                  = "StorageV2"
+  location                      = azurerm_resource_group.this.location
+  name                          = module.naming.storage_account.name_unique
+  resource_group_name           = azurerm_resource_group.this.name
+  min_tls_version               = "TLS1_2"
+  shared_access_key_enabled     = true
+  public_network_access_enabled = true
+  use_nested_nacl               = false
+  #use_nested_nacl              = true
```
Comment (jaredfholgate) [inline examples/private-endpoint-Staticip/main.tf:155]: Remove commented code

File: examples/private-endpoint-Staticip/main.tf [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+  managed_identities = {
+    system_assigned            = true
+    user_assigned_resource_ids = [azurerm_user_assigned_identity.example_identity.id]
+  }
+  tags = {
+    env   = "Dev"
+    owner = "John Doe"
+    dept  = "IT"
+  }
+  /*lock = {
```
Comment (jaredfholgate) [inline examples/private-endpoint-Staticip/main.tf:165]: Remove commented code

**Review by jaredfholgate — APPROVED** [review]
LGTM, just a few minor naming and tidying things

File: examples/private-endpoint-Staticip/main.tf (hunk 1 above)
Comment (chinthakaru) [reply examples/private-endpoint-Staticip/main.tf:101]: I will make this change with a different PR

**Pull request merged**